cg-spec/editors_draft.html

```diff
@@ -576,6 +542,40 @@ <h2>Concrete Syntaxes</h2>
     and the changes for the result set formats in
     <a href="#query-result-formats" class="sectionRef"></a>.</p>
 
+    <section id="annotation-syntax">
+      <h2>Annotation Syntax</h2>
+      <p>There are syntax additions for <a>quoted triples</a> and also for
+        <em>Annotation syntax</em> which provides a convenience shortcut in Turtle-star and
+        TriG-star syntaxes. It is used to both assert a triple and have that triple be the
+        subject of further triples. It uses delimiters `{|` and `|}` following an asserted
+        triple to make that triple, as a <a>quoted triple</a>, the subject of the enclosed
+        RDF triples.
```

Comment:
Changes made previously appear to have been lost before this step...  This sentence in particular was changed significantly and necessarily -- there are no "enclosed RDF triples"; there are "enclosed predicate + object pairs" (or however I phrased it previously).

Reply:
I've been suspicious that some suggestions were lost earlier, possibly because we are doing very fine grained edits and git only looks at lines.  Could you point out the changes because in the UI I see, I don't see them and also maybe close of other missed.

Here, I edited the latest github working copy as you can see from the commit.

Reply:
I have to hope they've all been caught.  I don't have a query that works against my brain to get the editing nits I've already picked...  (One downside to such a long-lived and much-massaged PR.)

Reply:
I've been going through things that weren't merged to re-surface them; there are a couple open for @afs to review. Still, I'm sure we'll find some way to further improve the document after merge.